docs/server_config/nginx.conf.dist

```diff
@@ -108,13 +108,21 @@ server {
         try_files $uri $uri/ /index.php?$args;
 
         # Images
+        rewrite ^/([0-9])(-[_a-zA-Z0-9-]*)?(-[0-9]+)?/.+2x.jpg$ /img/p/$1/$1$2$32x.jpg last;
         rewrite ^/([0-9])(-[_a-zA-Z0-9-]*)?(-[0-9]+)?/.+.jpg$ /img/p/$1/$1$2$3.jpg last;
```

Comment:
You can edit the current rule with  
```
rewrite ^/([0-9])(-[_a-zA-Z0-9-]*)?(-[0-9]+)?/.+(2x)?\.jpg$ /img/p/$1/$1$2$3.jpg last;
```
Can you also test with escaping the dot before jpg please? Could be a great improvement

Reply:
Aren't you redirecting to the classic image? I guess the purpose of this PR it to use the file ending with `2x.jpg`.

Reply:
Both are redirecting, he just add a `2x` before `\.jpg`, I just make it optional to prevent a long long list with the same regex :/